Android/Kotlin/source/Targets.cshtml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+  <ItemGroup>
+    @foreach (var art in @Model.MavenArtifacts) {
+    <AndroidJavaLibrary Include="$(MSBuildThisFileDirectory)..\..\jar\@(art.MavenGroupId).@(art.MavenArtifactId).jar" />
```

Comment:
Using the new `AndroidJavaLibrary` element so as to avoid embedding the jars in the dll.